### PR TITLE
[stable/elasticsearch-curator]add startingDeadlineSeconds cronjob spec

### LINKED ISSUE #21699: [stable/elasticsearch-curator] add startingDeadlineSeconds in cronjob spec
**Is your feature request related to a problem? Please describe.**
The current stable/elasticsearch-curator chart doesn't support  **startingDeadlineSeconds** spec in curator cronjob

**Describe the solution you'd like**
https://github.com/helm/charts/blob/master/stable/elasticsearch-curator/templates/cronjob.yaml
in **spec** block, add the followings:
```
  {{- with .Values.cronjob.startingDeadlineSeconds }}
  startingDeadlineSeconds: {{ . }}
  {{- end }}
```

### PR DESCRIPTION
**Pull request by ruccoon**
[stable/elasticsearch-curator]add startingDeadlineSeconds cronjob spec

Signed-off-by: ruccoon <ruccoon@hotmail.com>

#### What this PR does / why we need it:
The current stable/elasticsearch-curator chart doesn't support **startingDeadlineSeconds** spec in curator cronjob

#### Which issue this PR fixes
  - fixes #21699

https://github.com/helm/charts/issues/21699

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by ruccoon** [comment]
@desaintmartin @gianrubio could you help have a review, thx

File: stable/elasticsearch-curator/README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ their default values.
 | `cronjob.labels`                     | Labels to add to the cronjob                                | {}                                           |
 | `cronjob.concurrencyPolicy`          | `Allow\|Forbid\|Replace` concurrent jobs                    | `nil`                                        |
 | `cronjob.failedJobsHistoryLimit`     | Specify the number of failed Jobs to keep                   | `nil`                                        |
+| `cronjob.startingDeadlineSeconds`    | Secondes until now the controller counts the missed jobs    | `nil`                                        |
```
Comment (desaintmartin) [inline stable/elasticsearch-curator/README.md]:
```suggestion
| `cronjob.startingDeadlineSeconds`    | Optional deadline in seconds for starting the job if it misses scheduled time for any reason    | `nil`                                        |
```

**Review by desaintmartin — CHANGES_REQUESTED** [review]
Thanks for the PR!
/assign
/ok-to-test

File: stable/elasticsearch-curator/Chart.yaml [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+name: elasticsearch-curator
 sources:
 - https://github.com/kubernetes/charts/elasticsearch-curator
 - https://github.com/pires/docker-elasticsearch-curator
-maintainers:
-  - name: desaintmartin
-    email: cedric.dsm@gmail.com
-  - name: gianrubio
-    email: gianrubio@gmail.com
+version: 2.1.6
```
Comment (desaintmartin) [inline stable/elasticsearch-curator/Chart.yaml]: This is a new feature, could you bump minor version instead of patch?

**Review by desaintmartin — COMMENTED** [review]
Sorry, I completely forgot this one.

**Comment by desaintmartin** [comment]
/unassign
I removed myself from owners, @haad may be able to take over.

**Comment by ruccoon** [comment]
the feature has already been added in the stable version

closing PR

**Pull request closed without merging**